test/DifferentialExpressionMatrixSet_basic_test.py

```diff
@@ -82,27 +56,20 @@ def test_save_diff_exp_matrix_set_no_genome(
 
 
 def test_save_dem_set_mismatched_genomes(
-    test_data: dict,
     set_api_client: SetAPI,
     context: dict[str, str | list],
-    clients: dict[str, Any],
     ws_id: int,
+    diff_exp_matrix_mismatched_genome_refs: list[str],
 ) -> None:
     set_name = "dem_set_bad_genomes"
-    dem_set = {
-        "description": "this_better_fail",
-        "items": [
-            {
-                "ref": make_fake_diff_exp_matrix(
```

Comment:
all calls to `make_fake_***` have been moved into either the conftest or util files; the tests just use fixtures